summarize_pr.py

```diff
@@ -42,29 +40,113 @@ def summarize_pr(
                     f"Raw File Content:\n```\n{response.text}\n```\n\n"
                 )
 
+    return pull_request_content
+
+
+def _process_code_base(
+    pull_request_content: str, model_id: str, task_prompt: str, post_prompt: str = ""
+):
     model = GenerativeModel(
         model_id,
         system_instruction=[
             "You are an expert software engineer.",
         ],
+        generation_config=GenerationConfig(
```

Comment:
The use of the `GenerationConfig` object with a large `max_output_tokens` value is a good practice for accommodating potentially lengthy responses from the Gemini model. 

Consider adding a comment explaining the rationale for setting `max_output_tokens` to 8192. Additionally, it might be valuable to explore the possibility of adjusting this value based on the complexity of the input content or the desired response length.